src/sentry/demo/data/scen1b/python_transaction.json

```diff
@@ -0,0 +1,165 @@
+        ["Origin", "https://app.example.com"],
+        ["Referer", "https://app.example.com/toolstore"],
+        ["Sentry-Trace", "eaf14e4485864075aec039797501aca2-923dcc257d229405-1"],
+        [
+          "User-Agent",
+          "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.1 Safari/605.1.15"
+        ],
+        ["X-Cloud-Trace-Context", "8ae7c9108753b53d5e22f0f3d51efc42/12016370555220220073"],
+        ["X-Forwarded-For", ""],
+        ["X-Forwarded-Proto", "https"]
+      ],
+      "env": {"SERVER_NAME": "0.0.0.0", "SERVER_PORT": "8080"},
+      "inferred_content_type": "application/json"
+    },
+    "sdk": {
+      "name": "sentry.python",
+      "version": "0.19.2",
+      "integrations": [
+        "argv",
+        "atexit",
+        "dedupe",
+        "excepthook",
+        "flask",
+        "logging",
+        "modules",
+        "sqlalchemy",
+        "stdlib",
+        "threading"
+      ],
+      "packages": [{"name": "pypi:sentry-sdk", "version": "0.19.2"}]
+    },
+    "spans": [
+      {
+        "timestamp": 1615837032.413307,
+        "start_timestamp": 1615837032.40376,
+        "op": "db function: get inventory",
+        "span_id": "958d7b57cb3fc89b",
+        "parent_span_id": "9f629125fa90064c",
+        "trace_id": "eaf14e4485864075aec039797501aca2",
+        "same_process_as_parent": true
+      },
+      {
+        "timestamp": 1615837032.403948,
+        "start_timestamp": 1615837032.403814,
+        "op": "connect to db",
+        "span_id": "95245228f8f39673",
+        "parent_span_id": "958d7b57cb3fc89b",
+        "trace_id": "eaf14e4485864075aec039797501aca2",
+        "same_process_as_parent": true
+      },
+      {
+        "timestamp": 1615837032.410861,
+        "start_timestamp": 1615837032.40399,
+        "op": "run query",
+        "span_id": "a567b19a48a61708",
+        "parent_span_id": "958d7b57cb3fc89b",
+        "trace_id": "eaf14e4485864075aec039797501aca2",
+        "same_process_as_parent": true
+      },
+      {
+        "timestamp": 1615837032.410461,
+        "start_timestamp": 1615837032.404122,
+        "description": "SELECT * FROM inventory",
+        "op": "db",
+        "span_id": "b9ef6d1b42cb7ce5",
+        "parent_span_id": "a567b19a48a61708",
+        "trace_id": "eaf14e4485864075aec039797501aca2",
+        "same_process_as_parent": true
+      },
+      {
+        "timestamp": 1615837034.449384,
+        "start_timestamp": 1615837032.413422,
+        "op": "process order",
+        "span_id": "9186b2cdef32d090",
+        "parent_span_id": "9f629125fa90064c",
+        "trace_id": "eaf14e4485864075aec039797501aca2",
+        "tags": {"status": "ok"},
+        "same_process_as_parent": true
+      }
+    ],
+    "start_timestamp": 1615837032.402719,
+    "timestamp": 1615837034.460185,
+    "title": "checkout",
+    "transaction": "checkout",
+    "type": "transaction",
+    "user": {"email": "zllm@yahoo.com"},
```

Comment:
just noticed these have `@yahoo.com` and while we generate users later, should we change 'em to `blah@example.com` or something just to be on the safe side?

Reply:
@wmak these emails were auto-generated to begin with but I can just remove them in the template (we end up replacing it later anyways).